monitoring/helm/kube-prometheus/values.yaml

```diff
@@ -192,7 +192,7 @@ prometheus:
 
   ## External URL at which Prometheus will be reachable
   ##
-  externalUrl: ""
+  externalUrl: "https://alertmanager.apps.cloud-platform-live-0.k8s.integration.dsd.io"
```

Comment:
This is for `prometheus`, `alertmanager` is @ line 60.
I think we should do both?